test-lib.sh

```diff
@@ -63,7 +63,7 @@ function ct_enable_cleanup() {
 function ct_pull_image() {
   local image_name="$1"; shift
```

Comment:
```suggestion
  local image_name="$1"; [[ $# -gt 0 ]] && shift
```
For checking if there is some variable left to shift.